src/FiberTraces.cc

```diff
@@ -319,7 +396,7 @@ namespace pfsDRPStella = pfs::drp::stella;
       *varIt = sum(_trace->getVariance()->getArray()[i]);
       *maskIt = sum(_trace->getMask()->getArray()[i]);
     }
-    if ( !spectrum->setSpectrum( spec ) ){
+    if (!spectrum->setSpectrum(spec)){
```

Comment:
I think that this error condition is impossible.  If so, the check should be removed or replaced by an assertion.